File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/domain/interactors/types/UseCaseCompletable.java
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.domain.interactors.types;

import io.reactivex.Completable;

public interface UseCaseCompletable<Params> {

    Completable runUseCase(Params... params);
}

File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/domain/interactors/types/UseCaseCompletable.kt
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.domain.interactors.types

import io.reactivex.Completable

interface UseCaseCompletable<Params> {

    fun runUseCase(vararg params: Params): Completable
}
